fastlane/Fastfile

```diff
@@ -4,16 +4,18 @@ platform :macos do
   desc "Building Pusher"
   lane :ci do
     begin
-    sh "rm -rf ../pusher.app"
-    rescue => ex
-    UI.error(ex)
-    end
-    begin
-    sh "rm ../pusher.app.dSYM.zip"
+    sh "rm -rf ../PushTester.app;rm -rf ../PushTester.app.dSYM;"
```

Comment:
I thought `rm` with `force` option can't fail (event if the file doesn't exist) meaning begin-rescue is not necessary?

Reply:
Yes you're right! Fixed.